### ISSUE TITLE
previous known host file not found error

### ISSUE THREAD

**Issue by winggundamth** [ansible/ansible#5718]
I'm using Docker to create container and install ansible with pip install ansible.
I have no ansible.cfg and when start with simple command

```
ansible-playbook -vvvv -i /ansible-inventory /ansible.yaml
```

it always shows an error "previous known host file not found" in every ssh command. Please see details below

```
ansible 1.4.4
```

**ansible-inventory**

```
[webservers]
1.1.1.1:7001
```

**ansible.yaml**

```

---
- hosts: webservers
  remote_user: user
```

**output**

```
PLAY [webservers] ************************************************************* 

GATHERING FACTS *************************************************************** 
<1.1.1.1> ESTABLISH CONNECTION FOR USER: user
<1.1.1.1> EXEC ['ssh', '-tt', '-vvv', '-o', 'ControlMaster=auto', '-o', 'ControlPersist=60s', '-o', 'ControlPath=/.ansible/cp/ansible-ssh-%h-%p-%r', '-o', 'StrictHostKeyChecking=no', '-o', 'Port=7001', '-o', 'KbdInteractiveAuthentication=no', '-o', 'PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey', '-o', 'PasswordAuthentication=no', '-o', 'User=user', '-o', 'ConnectTimeout=10', '1.1.1.1', "/bin/sh -c 'mkdir -p $HOME/.ansible/tmp/ansible-1390374579.22-258448336777378 && chmod a+rx $HOME/.ansible/tmp/ansible-1390374579.22-258448336777378 && echo $HOME/.ansible/tmp/ansible-1390374579.22-258448336777378'"]
previous known host file not found
<1.1.1.1> REMOTE_MODULE setup 
<1.1.1.1> PUT /tmp/tmpZ3nNlw TO /var/www/.ansible/tmp/ansible-1390374579.22-258448336777378/setup
<1.1.1.1> EXEC ['ssh', '-tt', '-vvv', '-o', 'ControlMaster=auto', '-o', 'ControlPersist=60s', '-o', 'ControlPath=/.ansible/cp/ansible-ssh-%h-%p-%r', '-o', 'StrictHostKeyChecking=no', '-o', 'Port=7001', '-o', 'KbdInteractiveAuthentication=no', '-o', 'PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey', '-o', 'PasswordAuthentication=no', '-o', 'User=user', '-o', 'ConnectTimeout=10', '1.1.1.1', "/bin/sh -c '/usr/bin/python /var/www/.ansible/tmp/ansible-1390374579.22-258448336777378/setup; rm -rf /var/www/.ansible/tmp/ansible-1390374579.22-258448336777378/ >/dev/null 2>&1'"]
previous known host file not found
ok: [1.1.1.1]

PLAY RECAP ******************************************************************** 
1.1.1.1               : ok=1    changed=0    unreachable=0    failed=0   
```

I can ssh to that host with private key without problem or any error at all and I'm sure the host that I want to access has correctly information in known_hosts file. Also I still can use ansible without problem just error that annoy me.

One thing that I suspect is when I try command "cd ~" it's going to "/" but when I using ssh command it still read key file from "/root/.ssh/" and I already tried to copy "/root/.ssh/" directory to "/.ssh/" but it still shows error as above.

**Comment by mpdehaan** [comment]
Previously ansible did not look at the global known hosts file, this _may_ be resolved in the 1.5.X series

**Comment by jimi-c** [comment]
@winggundamth looking back through old issues, and it appears this was resolved some time ago. If you continue seeing any problems related to this issue, or if you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular issue is resolved.

Thank you!
